src/components/block/FooterCTACard/FooterCTACard.module.css

```diff
@@ -0,0 +1,28 @@
+.body {
+  color: var(--color-white);
+  font: var(--font-body-medium);
+  line-height: 1.3;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 15px;
+  max-width: 350px;
+  text-align: center;
+}
+
+.button {
+  margin-top: 30px;
+  text-align: center;
+}
+
+.container {
+  height: 400px;
+  width: 480px;
```

Comment:
You should not set the width of this component, since that will prevent any parent component from controlling the size of it. For example, if the parent allocated less space for this component, it would not be able to resize this component and this component would end up being cut off. See this example where I set the width of the parent element in Storybook to have a width of 320px:

<img width="658" alt="Screen Shot 2020-10-11 at 1 33 14 PM" src="https://user-images.githubusercontent.com/1002748/95689560-6cb28100-0bc6-11eb-89c4-f44f8c5236d8.png">